front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -0,0 +1,272 @@
+                  <TD aria-label="line-count" />
+                  <TD>{t('operationalPoint')}</TD>
+                  <TD>{t('code')}</TD>
+                  <TD>{t('endStop')}</TD>
+                  <TD>{t('startStop')}</TD>
+                  <TD>{t('motif')}</TD>
+                </TH>
+                {stdcmData?.simulation.base.stops.map(
+                  (stop, index) =>
+                    (index === 0 || stop.duration > 0) && (
+                      <TR key={index} style={styles.stopTableTbody}>
+                        <TD style={styles.stopTableIndexColumn}>{index + 1}</TD>
+                        <TD style={styles.stopTablePRColumn}>{stop.name || 'Unknown'}</TD>
+                        <TD style={styles.stopTableCHColumn}>{stop.ch}</TD>
+                        <TD style={styles.stopTableItalicColumn}>
+                          {index === stdcmData.simulation.base.stops.length - 1
+                            ? getStopTime(stop.time)
+                            : ''}
+                        </TD>
+                        <TD style={styles.stopTableStartColumn}>
+                          {index === 0 ? getStopTime(stop.time) : ''}
+                        </TD>
+                        <TD style={styles.stopTableItalicColumn}>
+                          {index === 0 || index === stdcmData.simulation.base.stops.length - 1
+                            ? t('serviceStop')
+                            : ''}
+                        </TD>
+                      </TR>
+                    )
+                )}
+              </Table>
+            </View>
+            <View style={styles.forBanner}>
+              <Text style={styles.forNumber}>n°987654</Text>
+              <View style={styles.forBox}>
+                <Text style={styles.for}>{t('for')}</Text>
+              </View>
+            </View>
+          </View>
+        </View>
+        <View style={styles.simulation}>
+          <View style={styles.simulationContainer}>
+            <Text style={styles.simulationUppercase}>{t('simulation')}</Text>
+            <Text style={styles.simulationLength}>
+              {Math.round((stdcmData?.path.length ?? 0) / 1000)}
+              {t('km')}
+            </Text>
+          </View>
+          <View style={styles.tableContainer}>
+            <Table style={styles.table}>
+              <TH style={styles.th}>
+                <TD aria-label="line-count" />
+                <TD>{t('operationalPoint')}</TD>
+                <TD>{t('code')}</TD>
+                <TD>{t('track')}</TD>
+                <TD>{t('endStop')}</TD>
+                <TD>{t('passageStop')}</TD>
+                <TD>{t('startStop')}</TD>
+                <TD>{t('weight')}</TD>
+                <TD>{t('referenceEngine')}</TD>
+                <TD>{t('conventionalSign')}</TD>
+                <TD>{t('crossedATE')}</TD>
+              </TH>
+              {stdcmData?.simulation.base.stops.map((stop, index) => (
```

Comment:
Use a real function and consts to simplify the code. Also rename stop to step if possible
```suggestion
              {stdcmData.simulation.base.stops.map((step, index) => {
                const isFirstStep = index === 0;
                const isLastStep = index === stdcmData.simulation.base.stops.length - 1;
                const prevStep = !isFirstStep ? stdcmData.simulation.base.stops[index - 1] : null;
                return (
                  <TR
```

Reply:
done